### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by camilo05**
Captura de Pantalla 2021-09-17 a la(s) 4 23 21 p  m

### CODE REVIEW

File: app/src/test/java/ar/com/wolox/android/example/ui/auth/login/AuthPresenterTest.kt [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    }
+
+    @Test
+    fun `When the password is empty`() = runBlocking {
+        presenter.onLoginButtonClicked(Extras.MockTesting.EMAIL, "").join()
+        val errors: List<LoginFormErrors> = listOf(LoginFormErrors.EMPTY_PASSWORD)
+
+        verify(view, times(1)).setErrors(errors)
+    }
+
```
Comment (rodrigo103) [inline app/src/test/java/ar/com/wolox/android/example/ui/auth/login/AuthPresenterTest.kt:63]: Maybe you could add two tests: one for when the password is invalid and one in which both email and password are invalid

**Review by rodrigo103 — COMMENTED** [review]
Excellent!! You are doing great Camilo. Keep it up!!

**Review by rodrigo103 — APPROVED** [review]

File: app/src/main/java/ar/com/wolox/android/example/utils/Extras.kt [hunk 2]
```diff
@@ -14,4 +15,15 @@ object Extras {
...
+    object MockTesting {
+        const val INVALID_EMAIL = "camilo.ortiz"
+        const val FAIL_EMAIL = "camilo.ortiz@wolox.co"
+        const val FAIL_PASSWORD = "123"
+        val FAIL_USERDATA = LoginData(FAIL_EMAIL, FAIL_PASSWORD)
+
+        const val EMAIL = "clinton.harris59@example.com"
+        const val PASSWORD = "123456"
+        val USERDATA = LoginData(EMAIL, PASSWORD)
+    }
```
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/utils/Extras.kt:28]: This is okay for now, but in real projects it's better to create a specific file for each tests' data.

**Review by ffasitella — APPROVED** [review]
Beautiful! Let's  go! 🙌🏽 ✨

**Pull request merged**